src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,41 @@
+.menu {
+  box-sizing: border-box;
+  height: 100vh;
+  background-color: #e5e6e8;
```

Comment:
It's recommended to use variables for colors to maintain consistency and ease of updates. Replace '#e5e6e8' with a variable, e.g., $menu-background-color.